mlfinlab/data_structures/base_bars.py

```diff
@@ -115,7 +154,7 @@ def _assert_csv(test_batch):
                   test_batch.iloc[0, 0])
 
     @staticmethod
-    def _update_high_low(high_price, low_price, price):
+    def _update_high_low(high_price: float, low_price: float, price: float) -> Union[float, float]:
```

Comment:
Would be better if update_high_low took only new price. Both high and low are already class variables and are accessible through self. (Alternatively you can change this method to the class one, than signature can be unchanged).